CustomImage/AzureConfig.ps1

```diff
@@ -0,0 +1,288 @@
+$logPath = "$PSScriptRoot\AzureConfig.ps1.log"
```

Comment:
Add copyright header:

# Copyright (c) Microsoft. All rights reserved.
# Licensed under the MIT license. See LICENSE file in the project root for full license information.